app/views/service_worker/index.js.erb

```diff
@@ -116,6 +116,7 @@
       '/search/reactions',
       '/search/tags',
       '/search/users',
+      '/search/usernames',
```

Comment:
Need for the service worker to ignore this endpoint and not try to wrap the JSON in HTML.